client/src/routes/[anime_id]/+page.server.ts

```diff
@@ -43,8 +44,50 @@ export const actions = {
         const title = form.get("title") as unknown as string;
         const image = form.get("image") as unknown as string;
 
-        favorites.set(mal_id, { title: title, image: image });
+        try {
+            const response = await api(`${SERVER_URL}/favorites`,
+                {
+                    method: 'POST',
+                    body: {
+                        id: mal_id.toString(),
+                        anime_id: mal_id.toString(),
+                        anime_title: title,
+                        anime_image_url: image,
+                        created: new Date().toISOString(),
+                        updated: new Date().toISOString(),
+                    }
+                }
+            );
 
-        return { success: true };
+            if (response.success) {
+                toast('Favorite added successfully!');
+            } else {
+                if (response.error === 'Maximum limit of 5 favorites reached') {
+                    toast('You have reached the limit of 5 favorites.');
+                } else {
+                    toast('An error occurred while adding the favorite.');
```

Comment:
This toast are not working, cos you are calling them from server :) So the browser doesnt know about them. To use toast on client side, in +page.svelte you need to listen to `form` element:

```
export let form;
$: if (form......) { toast() };
```